components/Header/index.stories.tsx

```diff
@@ -9,13 +9,16 @@ const meta: Meta = {
     (Story: () => ReactElement): JSX.Element => (
       <>
         <Story />
-        <MockMain />
+        <MockBlock>Section 1</MockBlock>
+        <MockBlock>Section 2</MockBlock>
+        <MockBlock>Section 3</MockBlock>
+        <MockBlock>Section 4</MockBlock>
       </>
     ),
   ],
   parameters: {
     a11y: {
-      element: '#root *:not(main, main *)',
+      element: '#root *:not(.mockBlock, .mockBlock *)',
```

Comment:
Same as the other stories file with regard to the `:not()` pseudo-class 😁

Reply:
Cool. Thanks for the trick to select a class name with a wildcard.